ajax/libs/iGrowl/package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "iGrowl",
+	"filename": "igrowl.min.js",
+	"description": "A lightweight jQuery plugin that generates growl-like notifications with an emphasis on icons.",
+	"homepage": "https://catc.github.io/iGrowl/",
+	"author": {
+		"name": "Catalin Covic",
+		"email": "catalin.covic@gmail.com",
+		"url": "https://devpost.com/catcc"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/catc/iGrowl"
+	},
+	"keywords": [
+		"iGrowl",
+		"notifications",
+		"alerts"
+	],
+	"license": "MIT",
+	"autoupdate": {
+		"source": "git",
+		"target": "git://github.com/catc/iGrowl.git",
+		"fileMap": [{
+			"basePath": "dist",
+			"files": [
+				"**/*"
```

Comment:
Because it will fetch `*.json` files by using `"**/*"`, let's use `"**/*.!(json)"` instead.